test_tools/scripts/record_video_from_camera.py

```diff
@@ -31,15 +32,25 @@ def callback(self, data):
     def clean_shutdown(self):
         if self.video_writer is not None:
             self.video_writer.release()
-        rospy.loginfo(f"Saving video file {self.file_name}")
+            rospy.loginfo(f"Saving video file {self.file_name}")
+        else:
+            rospy.loginfo(f"No video saved, no image received on topic")
 
     def __del__(self):
         self.clean_shutdown()
 
 
 def main():
+    image_topic = "/hero/head_rgbd_sensor/rgb/image_raw"
```

Comment:
I think remapping might be better for this.

Reply:
ease of use is more important to me in this case. you can still specify another topic if needed.

Reply:
Making it too easy too use, will make people lazy. And people will not understand anymore what is happening under the hood. Keep that in mind.

Reply:
There is no need to know what is happening under the hood. As long as the documentation is up to date. (I added an argparser to improve on the previous version for this.)